### PR TITLE
jsonnet/mixin: add PrometheusOperatorRejectedResources alert

### PR DESCRIPTION
**Pull request by simonpasquier**
Follow-up of #3421

### CODE REVIEW

File: example/mixin/alerts.yaml [hunk 1]
```diff
@@ -61,3 +61,14 @@ groups:
     for: 5m
     labels:
       severity: warning
+  - alert: PrometheusOperatorRejectedResources
+    annotations:
+      description: Prometheus operator in {{ $labels.namespace }} namespace rejected
+        {{ printf "%0.0f" $value }} {{ $labels.controller }}/{{ $labels.resource }}
+        resources.
+      summary: Resources rejected
```
Comment (paulfantom) [inline example/mixin/alerts.yaml:69]: nit: should we mention that this is about prometheus-operator resources in summary?

**Review by paulfantom — APPROVED** [review]
lgtm, just one nit

File: example/mixin/alerts.yaml (hunk 1 above)
Comment (simonpasquier) [reply example/mixin/alerts.yaml:69]: yes sounds good to me

**Comment by s-urbaniak** [comment]
LGTM, thank you!

**Pull request merged**